README.md

```diff
@@ -136,6 +131,8 @@ From the `Sources` list, select and drag the `Sample Data` operator onto the can
 
 ![add-sample-data-source](doc/source/images/add-sample-data-source.png)
 
+> NOTE: As you've probably already noticed, red error indicators tell you when required settings are missing. For example, some settings are required. You will also see that some operators require a source and/or target connection. When an operator has a red spot on it, you can hover over it to see what the errors are.
```

Comment:
in above text - "that can be drag-n-dropped"
"the operators are divided into types"